### PR TITLE
Fixes/navbar

### PR DESCRIPTION
**Pull request by ryanmao725**
Attempting to address #293

### CODE REVIEW

File: src/NavBar.jsx [hunk 1]
```diff
@@ -108,7 +108,8 @@ class NavBar extends Component {
         let navLinks = [];
         for (let i = 0; i < keys.length - 1; i++) {
             navLinks.push(
-                <NavItem key={i}>
+                <NavItem key={i}
+                    className={keys[i] === "ABOUT" && window.innerWidth < 768 ? "pt-3" : ""}>
```
Comment (thekabistro) [inline src/NavBar.jsx]: Make this more generic by using  i=0?   Its not always the about page at the top of the drawer

**Review by thekabistro — CHANGES_REQUESTED** [review]

File: src/NavBar.jsx [hunk 2]
```diff
@@ -108,7 +108,8 @@ class NavBar extends Component {
         let navLinks = [];
         for (let i = 0; i < keys.length - 1; i++) {
             navLinks.push(
-                <NavItem key={i}>
+                <NavItem key={i}
+                    className={i = 0 && window.innerWidth < 768 ? "pt-3" : ""}>
```
Comment (shivanmodha) [inline src/NavBar.jsx]: `i === 0`

**Review by shivanmodha — CHANGES_REQUESTED** [review]
Doesn't appear to work here:
image

File: src/NavBar.jsx (hunk 2 above)
Comment (ryanmao725) [reply src/NavBar.jsx]: ah fuck

Comment (shivanmodha) [reply src/NavBar.jsx]: xD

**Comment by shivanmodha** [comment]
Can you get this to work with the home as well?

**Review by shivanmodha — APPROVED** [review]
LGTM! 4th times the charm ay?

**Pull request merged**